### ISSUE TITLE
We should have a version number

### ISSUE THREAD

**Issue by tleylan** [beefsweater/VRLX.Website#39]
I'd like to see us version the website. It should be subtle, it doesn't even have to be visible to everyone but we should have an incrementing version number (each time we release) and a simple way to know which version we currently have running.

@josempaii @beefsweater

**Comment by beefsweater** [comment]
I agree. you can put it wherever you would like. I don't mind it being on the bottom of the page, either.

**Comment by tleylan** [comment]
The "trick" is how to maintain the version number. I haven't found a good way for a static website. Nothing enforces a rule that a version number needs to be updated and nothing stops someone (with access) from editing the website directly. If you find a way to do it let's discuss it.
